mpf/platforms/base_serial_communicator.py

```diff
@@ -64,7 +65,10 @@ async def _connect_to_hardware(self, port, baud, xonxoff=False):
 
         serial = self.writer.transport.serial
         if hasattr(serial, "set_low_latency_mode"):
-            serial.set_low_latency_mode(True)
+            try:
+                serial.set_low_latency_mode(True)
+            except NotImplementedError:
+                self.log.debug("Could not set %s to low latency mode.", port)
```

Comment:
I would go with info or warning here. This should work for FTDI at least. Might be a permission issue (i.e. only works as root)

Reply:
I was getting an exception on my M1 Mac for this, which was the only reason I added the handler. The serial descriptions says it is supported but it fails when actually set.

Reply:
I think somebody from the OPP folks saw that error on Linux as well. They said it worked when they ran MPF with sudo but not as user. So it is probably a good idea not to crash here but show a notice. I think they got some OSError "Not supported" or something totally confusing.

Reply:
I thought that's what it was doing, but I'm not familiar with Python exception handling. Do you mean it should be using a different message than the log.debug() ? with the except: block it runs ok on my machine where the exception was halting it previously.